zhaquirks/tuya/ts000x.py

```diff
@@ -83,6 +83,93 @@ class Switch_1G_GPP(CustomDevice):
         },
     }
 
+class Switch_2G_GPP(CustomDevice):
+    """Tuya 2 gang switch module with restore power state support."""
+
+    signature = {
+        MODEL: "TS0003",
```

Comment:
The model is TS0002

Reply:
Thanks fixed !!
(Its the "corrupted clipboard buffer syndrome" = Copy and paste errors in the machine)